GPT/beta-commands/beta-gpt.talon

```diff
@@ -6,7 +6,7 @@ tag: user.gpt_beta
 model find <user.text>: user.gpt_find_talon_commands(user.text)
 
 # Using the context of the text on the clipboard, update the selected text
-model blend [from] clip:
+model blend clipboard:
```

Comment:
Should this just be "clip"?

Reply:
Sure feel free to edit any model blend commands in the other PR. I think that is reasonable. Model clip follows the pattern